ztp/policygenerator/policyGen/policyBuilder.go

```diff
@@ -86,6 +87,25 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 					if err != nil {
 						return policies, err
 					}
+					//set to default remediationAction
+					remediationActionVal := acmPolicy.Spec.RemediationAction
+					if policyGenTemp.Spec.RemediationAction != "" {
```

Comment:
this will be always true policyGenTemp.Spec.RemediationAction default is enforce.